suite/tests/CMakeLists.txt

```diff
@@ -2646,6 +2646,10 @@ if (NOT ANDROID) # TODO i#38: Port test to Android.
     torunonly_ci(client.attach_blocking linux.infloop client.attach_test.dll
       client-interface/attach_blocking.runall "" "" "")
   endif ()
+  if (LINUX AND X64)
+    # TODO i#7046: Verify the content of the output file.
```

Comment:
This comment seems like it should be inside the test file, not here as there would be no change made here to add the verification.

Reply:
I've moved it to the test file.